.eslintrc.json

```diff
@@ -7,6 +7,9 @@
     "parserOptions": {
         "sourceType": "module"
     },
+    "globals": {
+        "console": true
```

Comment:
Hmm, do we want to allow `console` everywhere or just in the specific places we expect to use it?

Reply:
I can also add `/* global console */` in those two files if you think it's better?